common/txmgr/broadcaster.go

```diff
@@ -475,7 +475,7 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) hand
 	}
 
 	lgr := etx.GetLogger(logger.With(eb.lggr, "fee", attempt.TxFee))
-	lgr.Infow("Sending transaction", "txAttemptID", attempt.ID, "txHash", attempt.Hash, "meta", etx.Meta, "feeLimit", etx.FeeLimit, "attempt", attempt, "etx", etx)
+	lgr.Infow("Sending transaction", "txAttemptID", attempt.ID, "txHash", attempt.Hash, "meta", etx.Meta, "feeLimit", attempt.ChainSpecificFeeLimit, "callerProvidedFeeLimit", etx.FeeLimit, "attempt", attempt, "etx", etx)
```

Comment:
The behaviour has changed. `feeLimit` now is represented by `ChainSpecificFeeLimit` instead of `etx.FeeLimit` .

Reply:
That's what I'm trying to distinguish in the updated logs since etx.feeLimit is the client set fee limit, is this incorrect?

Reply:
@dimriou The reason for this change is we were logging a misleading value for `feeLimit`. The fee limit we actually broadcast with is `attempt.ChainSpecificFeeLimit` which could be different than the `etx.FeeLimit` because of `LimitMulitplier`.

Reply:
All good, just wanted to confirm that that was the intended behaviour.